packages/design-tokens/package.json

```diff
@@ -21,11 +21,12 @@
   "private": false,
   "license": "MIT",
   "sideEffects": false,
+  "type": "module",
   "scripts": {
     "test": "FORCE_COLOR=1 jest",
     "test:ci": "pnpm test -- --ci",
     "build": "pnpm clean && pnpm prepublishOnly",
-    "build:json": "pnpm ts-node --files ./bin/buildCssVarTokens.ts",
+    "build:json": "tsx ./bin/buildCssVarTokens.ts",
```

Comment:
`ts-node` decided to stop working for some reason, switched to the better `tsx`

Reply:
good choice, we should switch to `tsx` everywhere. `ts-node` is full of problems